code/functions.jl

```diff
@@ -0,0 +1,166 @@
+"""
+TODO
+"""
+function generate(x, σ)
+    @assert σ >= 0.0
+    return σ > 0.0 ? rand(TruncatedNormal(x,σ, 0.0, 100.0)) : x
+end
+
+"""
+***Simulations for the host population and specialist parasitoid population dynamics for the first 50 generations***
+
+- `N`: Initial host population size
+- `P`: Initial parasitoid population size
+- `t`: Number of generations
+- `f`: Type of parasite chosen to to the simulations (specialist or generalist)
+- `F`: Finite rate of increase of the host population
+- `D`: Density independent mortality (as a probability of survival)
+- `c`: Number of parasitoids emerging from each host parasitized
+- `a`: Searching efficiency (per capita)
+- `h`: Saturation number of parasitoids
+- `b`: Rate of appoaching h
+- `th`: Handling time (as a proportion of the total time)
+- `m`: Extent of clumping of the parasitoid attacks
+
+Return : `dynamics`: Matrix containing the host and parasite density for the initial populations and for every generation simulated
```

Comment:
should say the row/column structure of the return (i.e. time is rows).

Reply:
Fixed.